documentation/library_guide/parallel_api_main.rst

```diff
@@ -5,15 +5,15 @@ Parallel API is an implementation of the C++ standard libraries algorithms and e
 policies, as specified in the ISO/IEC 14882:2017 standard (commonly called C++17). The implementation
 supports the unsequenced execution policy and the ``shift_left``/``shift_right`` algorithms, which are specified
 in the final draft of the C++ 20 standard (N4860). For more details see the `C++ Standard Execution
-Policies <https://en.cppreference.com/w/cpp/algorithm/execution_policy_tag_t>`_. |onedpl_long| provides
-specific versions of the algorithms, including:
+Policies <https://en.cppreference.com/w/cpp/algorithm/execution_policy_tag_t>`_. |onedpl_long| (|onedpl_short|)
+provides specific versions of the algorithms, including:
 
 * Segmented reduce
 * Segmented scan
 * Vectorized search algorithms
 
 Parallel API offers support for the parallel and vectorized execution of
-algorithms on Intel® processors and heterogeneity support with |dpcpp_short| execution policies.
+algorithms on Intel® processors and heterogeneity support with SYCL* execution policies.
```

Comment:
Execution policies are not a part of SYCL specification; I do not think we should use "SYCL [execution] policies" in the documentation. Let's use "device [execution] policies" instead, consistent with the name of the class. In this particular case, we can say e.g. "... and heterogeneity support with a DPC++ based implementation for device execution policies".

Reply:
Updated.